package.json

```diff
@@ -47,7 +47,7 @@
     "@ledgerhq/hw-transport-http": "^4.66.0",
     "@ledgerhq/hw-transport-node-hid": "^4.66.0",
     "@ledgerhq/hw-transport-node-hid-singleton": "^4.66.0",
-    "@ledgerhq/ledger-core": "^3.0.0",
+    "@ledgerhq/ledger-core": "^3.0.0-beta.6",
```

Comment:
this should not happen. it should be 3.0.0

Reply:
🤦‍♂

Reply:
you got it